src/search.js

```diff
@@ -27,7 +32,7 @@ export function addNormalizedDescription(propertyName, objects) {
   return map(
     object => assoc(
       'normalizedDescription',
-      has(propertyName, object)
+      object[propertyName]
```

Comment:
This is not about code style.

If object **has** a `propertyName` property, but that the value of this property is `null`, then we dont wan't to remove the diacritics, as it would cause an error.

We need this change because with the new API, a parameter description may be null.